services/gateway/endpoints/src/main/java/org/eclipse/ditto/services/gateway/endpoints/routes/cloudevents/UnsupportedDataSchemaException.java

```diff
@@ -0,0 +1,129 @@
+ /*
+  * Copyright (c) 2020 Contributors to the Eclipse Foundation
+  *
+  * See the NOTICE file(s) distributed with this work for additional
+  * information regarding copyright ownership.
+  *
+  * This program and the accompanying materials are made available under the
+  * terms of the Eclipse Public License 2.0 which is available at
+  * http://www.eclipse.org/legal/epl-2.0
+  *
+  * SPDX-License-Identifier: EPL-2.0
+  */
+ package org.eclipse.ditto.services.gateway.endpoints.routes.cloudevents;
+
+ import java.net.URI;
+ import java.text.MessageFormat;
+ import java.util.Set;
+
+ import javax.annotation.Nullable;
+ import javax.annotation.concurrent.Immutable;
+ import javax.annotation.concurrent.NotThreadSafe;
+
+ import org.eclipse.ditto.json.JsonObject;
+ import org.eclipse.ditto.model.base.common.HttpStatusCode;
+ import org.eclipse.ditto.model.base.exceptions.DittoRuntimeException;
+ import org.eclipse.ditto.model.base.exceptions.DittoRuntimeExceptionBuilder;
+ import org.eclipse.ditto.model.base.headers.DittoHeaders;
+ import org.eclipse.ditto.model.base.json.JsonParsableException;
+
+ /**
+  * Thrown if a request with an unsupported data schema is made.
+  */
+ @Immutable
+ @JsonParsableException(errorCode = UnsupportedDataSchemaException.ERROR_CODE)
+ public final class UnsupportedDataSchemaException extends DittoRuntimeException {
```

Comment:
Please move the exception to package `org.eclipse.ditto.model.base.exceptions` where also the `UnsupportedMediaTypeException` is.
The reason why unit tests fail is because of added exceptions in this module.

Please also rename to `CloudEventUnsupportedDataSchemaException` as this is a "cloud events" specific error.